tests/framework/utils/helm_helper.go

```diff
@@ -55,12 +56,23 @@ func (h *HelmHelper) InstallLocalRookHelmChart(chartName string, deployName stri
 	if namespace != "" {
 		cmdArgs = append(cmdArgs, "--namespace", namespace)
 	}
-	_, err := h.Execute(cmdArgs...)
+	result, err := h.Execute(cmdArgs...)
```

Comment:
why if you use a retry loop? then you can change the number of retries easily and the code on each retry is the same

Reply:
I don't think we want to retry when a known/legit error occurs.  In certain cases we will want the test to fail right away. I like adding retries on functions using this method.